package-lock.json

```diff
@@ -1,8 +1,27 @@
 {
   "name": "hwf-publicapp",
   "version": "0.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
```

Comment:
I just check the whole change in this file and it looks like you downgraded the govuk front end and it's referenced twice.

Reply:
resolved